.eslintrc

```diff
@@ -46,7 +46,7 @@
     "import/no-named-as-default": 0,
     "no-underscore-dangle": 0,
     "no-empty-pattern": 0,
-    "object-shorthand": 0,
+    "object-shorthand": 2,
```

Comment:
Is `2` the default? We want to enable rules by removing their entries from the dot file, rather than hard-coding their settings.

Reply:
'Always' is the default, but i have changed it to only 'properties'